testing/supporting_docs/appium_information.md

```diff
@@ -0,0 +1,69 @@
+## Appium 
+Appium is a cross platform UI testing tool for mobile applications running on 
+either android or iOS devices.  It can be used to test native applications, 
```

Comment:
@twnolan consistency should be persisted with the word 'android' across the whole document, e.g. starting with either lower or upper case.